link/src/main/java/com/stripe/android/link/analytics/LinkEventsReporter.kt

```diff
@@ -4,16 +4,22 @@ import androidx.annotation.RestrictTo
 
 @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
 internal interface LinkEventsReporter {
+    fun onInvalidSessionState(state: SessionState)
+
     fun onInlineSignupCheckboxChecked()
     fun onSignupStarted(isInline: Boolean = false)
     fun onSignupCompleted(isInline: Boolean = false)
-    fun onSignupFailure(isInline: Boolean = false)
-    fun onAccountLookupFailure()
+    fun onSignupFailure(isInline: Boolean = false, error: Throwable)
+    fun onAccountLookupFailure(error: Throwable)
```

Comment:
Can we narrow down the error type here?

Reply:
They are mainly `StripeException` types. I've added an extension property to handle the messages.